src/Sulu/Bundle/MediaBundle/Media/Manager/MediaManager.php

```diff
@@ -197,6 +199,7 @@ public function __construct(
         $this->tokenStorage = $tokenStorage;
         $this->securityChecker = $securityChecker;
         $this->ffprobe = $ffprobe;
+        // TODO permissions are deprecated and should be removed in 2.3
```

Comment:
@Prokyonn can you create a PR for this? i am afraid we will forget about it if we dont do that now 🙈